### PR TITLE
Fixes transformation viruses yeeting you out of the game

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Stops transformation viruses from throwing you out of the game if you transform

## Why It's Good For The Game
Code should work and keep you playing without issues

## Testing
Ate xenomicrobes, became hunter

## Changelog
:cl:
fix: Fixed transformation viruses stopping you from playing the game
/:cl:

### CODE REVIEW

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -67,10 +67,10 @@
 		affected_mob.create_log(MISC_LOG, "has transformed into [new_mob], due to having the virus \"[src]\"")
 		if(istype(new_mob))
 			new_mob.a_intent = "harm"
-			if(affected_mob.mind)
-				affected_mob.mind.transfer_to(new_mob)
 			if(isrobot(new_mob))
 				new_mob.rename_self("Cyborg", TRUE, TRUE)
```
Comment (Contrabang) [inline code/datums/diseases/transformation.dm:71]: IMO, this should be changed to be after, otherwise I don't think theyll get the prompt, right?

Comment (DGamerL) [reply code/datums/diseases/transformation.dm]: Good point, changed!

Comment (DGamerL) [reply code/datums/diseases/transformation.dm]: Tested both xenomicrobes and nanomachines, both worked as intended

**Review by S34NW — APPROVED** [review]

**Comment by PIerreLouisH** [comment]
Opsie.

**Pull request merged**